src/conditional-flow/multiple-conditions.js

```diff
@@ -2,18 +2,18 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
-  // TODO: write code in this function body to pass the tests
-
+  if (num >= lower && num <= upper) {
```

Comment:
This could be simplified to 
```js
return num >= lower && num <= upper
```

Reply:
I have learnt about this just this morning, Definitely something I will try to implement in the next exercises